app/models/auth_probe_result2.rb

```diff
@@ -3,15 +3,17 @@
 # Represents IIIF Auth V2 probe service response
 # See https://iiif.io/api/auth/2.0/#probe-service-response
 class AuthProbeResult2
-  def initialize(status:, note: nil)
+  def initialize(status:, note: nil, heading: nil)
```

Comment:
We only use note an heading on error results, so can these fields move/remain on `ErrorResult`?

Reply:
And then not_found and bad_request become ErrorResults instead of AuthProbeResult2

Reply:
Ah, I thought they were already.  Nevermind then. `ErrorResult` was perhaps too general of a name and it ought to have been `NotAuthorizedResponse`